tests/foreman/ui/test_jobinvocation.py

```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Acceptance
 
-:CaseComponent: UI
+:CaseComponent: WebUI
```

Comment:
Earlier comment about interface component applies.

I think this one should be `RemoteExecution`? Seems to be closest match here. But you can double-check with someone more familiar with the feature.